### PR TITLE
add recipes for EvtGen installation in O2

### PR DESCRIPTION
**Pull request by ffionda**
recipes for building EVEGEN, photos, tauola (authors: preghenella. ffionda, jbiernatKRK)

### CODE REVIEW

File: evtgen.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - alibuild-recipe-tools
+  - "GCC-Toolchain:(?!osx)"
+env:
+   EVTGENDATA: "$EVTGEN_ROOT/share"
+---
+#!/bin/bash -e
+rsync -a --delete --exclude '**/.git' $SOURCEDIR/ ./
+
+# adjust the configure scripts
+#sed -i -e "s/FLIBS=\"-lfrtbegin -lf2c\"$/FLIBS=\"-lgfortran -lgfortranbegin\"/" configure
```
Comment (preghenella) [inline evtgen.sh:20]: @ffionda 
please remove this commented line (my bad)

File: evtgen.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+sed -i -e "s/FLIBS=.*$/FLIBS=\"-lgfortran\"/" configure
+sed -i -e "s/PYTHIALIBLIST=.*$/PYTHIALIBLIST=\"-lpythia8 -lpythia8lhapdf6\"/" configure
+#sed -i -e "s/LIBDIRLIST=.*$/LIBDIRLIST=\"\"/" configure
```
Comment (preghenella) [inline evtgen.sh:23]: and this one as well (again my bad)

**Comment by preghenella** [comment]
hi @TimoWilken @ktf 
do you know why the checks are not progressing on this PR?

**Comment by TimoWilken** [comment]
Hi @preghenella. @ffionda is a "first-time contributor" to this repository, which means that this PR must be approved by someone before CI can run (this is for security reasons).

If you'd like the CI to run, you first have to give the PR an approving review on GitHub.

**Review by preghenella — APPROVED** [review]

**Comment by preghenella** [comment]
I see, thanks @TimoWilken

**Comment by preghenella** [comment]
failure of the tests seems unrelated to this PR.
@TimoWilken @ktf can we merge it?

**Comment by TimoWilken** [comment]
Seeing as this PR doesn't change existing recipes and CI failures are unrelated, it's fine to merge in my opinion.

**Review by ktf — APPROVED** [review]

**Pull request merged**